ci-operator/step-registry/baremetalds/sno/gather/baremetalds-sno-gather-commands.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds sno gather command ************"
+
+# Ensure our UID, which is randomly generated, is in /etc/passwd. This is required
+# to be able to SSH.
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    else
+        echo "/etc/passwd is not writeable, and user matching this uid is not found."
+        exit 1
+    fi
+fi
```

Comment:
You could add a `fix_uid.sh` in your image to reduce the differences from the `devscripts` (and so the differences within this step)

Reply:
fixed

Reply:
Sorry, reverting until we internally resolve the issue in our image